### PR TITLE
18 apropos

### PR DESCRIPTION
**Pull request by berenice89**

### CODE REVIEW

**Comment by berenice89** [comment]
Resolved conflicts in app.jsx

**Review by YannYves — DISMISSED** [review]
Looks alright to me !

File: src/components/BurgerMenu.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const StyledBurger = styled.button`
+  position: absolute;
+
+  /* top: 4%;
+  left: 2rem; */
```
Comment (LoicBrassart) [inline src/components/BurgerMenu.jsx]: Dead code

File: src/components/BurgerMenu.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    outline: none;
+  }
+
+  div {
+    width: 2rem;
+    height: 0.25rem;
+    background: ${({ open }) => (open ? '#76d03a' : '#0099b2')};
+    border-radius: 10px;
+    transition: all 0.3s linear;
+    /*position: relative;*/
```
Comment (LoicBrassart) [inline src/components/BurgerMenu.jsx]: Dead code

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
You should consider *never* commenting code in the first place, to avoid forgetting deleting it later

**Review by YannYves — DISMISSED** [review]
Dead code removed

**Comment by YannYves** [comment]
ESLint check failed . Please review .

**Review by YannYves — CHANGES_REQUESTED** [review]
ESLint Check failed, please review .

**Review by YannYves — DISMISSED** [review]
ok

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by YannYves — APPROVED** [review]
ok to me

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**